frontend/src/pages/Home.tsx

```diff
@@ -15,7 +15,7 @@ export function Home() {
       <Helmet>
         <title>Home | USHS Housing Portal</title>
       </Helmet>
-      <h1>This is the home page.</h1>
+      <h1>This is the home page. </h1>
```

Comment:
```suggestion
      <h1>This is the home page.</h1>
```

Removing this space should allow this file to merge cleanly with main